bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/dto/NAThing.java

```diff
@@ -55,13 +54,16 @@ public ModuleType getType() {
     public boolean isReachable() {
         // This is not implemented on all devices/modules, so if absent we consider it is reachable
         Boolean localReachable = this.reachable;
-        boolean result = localReachable != null ? localReachable : true;
-        // and we double check by comparing data freshness
+        return localReachable != null ? localReachable : true;
+    }
+
+    public boolean hasFreshData(int dataFreshnessLimit) {
+        // check by comparing data freshness
         ZonedDateTime localLastSeen = lastSeen;
```

Comment:
Would suggest `Instant` also here.

Reply:
If data display was properly switched automatically to user's browser local or using TimeZoneProvider, I would switch all the DTO of the binding to Instant. I saw the discussion [here](https://github.com/openhab/openhab-core/issues/2898). But doing that, I'll force users to overload each and every timestamp displayed to have something immediately meaningfull to them. I prefered take in account the TimeZoneProvider when deserializing. I think at this stage it's better to leave it like this until the discussion in core has reached a consensus.

Reply:
I agree if this is mapped to DateTimeType, I probably assumed that this was only local timestamp for determining data age. Please resolve comment if this is indeed not the case.